src/commands/synthetics/__tests__/run-test.test.ts

```diff
@@ -395,20 +273,20 @@ describe('run-test', () => {
 
     test('should default to datadog us api', async () => {
       process.env = {}
-      const command = new RunTestCommand()
-      await command['resolveConfig']()
 
-      expect(command['getDatadogHost']()).toBe('https://api.datadoghq.com/api/v1')
-      expect(command['getDatadogHost'](true)).toBe('https://intake.synthetics.datadoghq.com/api/v1')
+      expect(runTests.getDatadogHost(false, config)).toBe('https://api.datadoghq.com/api/v1')
+      expect(runTests.getDatadogHost(true, config)).toBe('https://intake.synthetics.datadoghq.com/api/v1')
     })
 
     test('should be tunable through DATADOG_SITE variable', async () => {
       process.env = {DATADOG_SITE: 'datadoghq.eu'}
-      const command = new RunTestCommand()
-      await command['resolveConfig']()
 
-      expect(command['getDatadogHost']()).toBe('https://api.datadoghq.eu/api/v1')
-      expect(command['getDatadogHost'](true)).toBe('https://api.datadoghq.eu/api/v1')
+      expect(runTests.getDatadogHost(false, {...config, datadogSite: process.env.DATADOG_SITE as string})).toBe(
+        'https://api.datadoghq.eu/api/v1'
+      )
+      expect(runTests.getDatadogHost(true, {...config, datadogSite: process.env.DATADOG_SITE as string})).toBe(
+        'https://api.datadoghq.eu/api/v1'
+      )
```

Comment:
In these tests, I guess we should not pass `datadogSite`, since we expect it to be overriden by the env variable `DATADOG_SITE` anyway, no?

Reply:
Done!
Have refactored the `getDatadogHost` tests : I've added a test for the env var `DD_API_HOST_OVERRIDE` and have  removed the test for `DATADOG_SITE` as this is a `resolveConfig()`  functionality which is tested in `cli.test.ts`